src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts

```diff
@@ -172,22 +176,56 @@ class ZWaveJSConfigDashboard extends LitElement {
                         )}
                       </mwc-button>
                     </a>
-                    <mwc-button @click=${this._addNodeClicked}>
+                  </div>
+                </ha-card>
+                <ha-card header="Diagnostics">
```

Comment:
The action items (other than dump) are not really diagnostics actions.
Do we even need a header for this card?

Reply:
Would you heal your network or re-configure the server if you're not in trouble?

Reply:
No, but that is more "Troubleshooting" or "Management" and not "Diagnostics"?

Reply:
Ah yes. 

@matthiasdebaat what do you think ?

Reply:
I think this is fine (at least for now)